frontend/src/routes/dashboard/index.tsx

```diff
@@ -31,12 +32,19 @@ const Dashboard = () => {
                 Root
             </Button>
             <br/>
-
-            <br/>
-
             <Button.Link to="/">
                 Trunk
             </Button.Link>
+            <br/>
+            <Sidebar>
```

Comment:
It's fine to keep this here for now, but if you want something quick to do before we catch up next, see if you can move this into the `routes/__root.tsx` file so that it is visible on every route

Reply:
Since this is now going to show on every page, if we go to the dashboard, I assume that two sidebars would show, can you please remove this sidebar